packages/api-server/db_migrations/001_create-init.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE IF NOT EXISTS users (
```

Comment:
tables look good over all for users we have the join table, could still FK it, but we will just inner or left join so it does not matter. 

The portals tables the same as before?

Reply:
> The portals tables the same as before?

They are slightly different.